include/malloc.h

```diff
@@ -5,6 +5,12 @@
 #define calloc(ct, sz) AllocZeroed((ct) * (sz))
 #define free Free
 
+#define FREE_AND_SET_NULL(ptr)          \
```

Comment:
Is this macro necessary?

Reply:
Yes, it's used basically everywhere when memory is freed.